ci/build_wheel.sh

```diff
@@ -5,6 +5,7 @@ set -euo pipefail
 
 package_name=$1
 package_dir=$2
+package_type=$3
```

Comment:
I guess this is in anticipation of the C++ wheel? I have a very minor preference for sticking that change in the subsequent PR to make the reason more obvious, but now that it's done let's leave it here.

Reply:
Technically yes, but I think it'd be desirable even if we weren't adding a C++ wheel in this repo.

In my mind, https://github.com/rapidsai/shared-workflows/pull/209 and https://github.com/rapidsai/gha-tools/pull/105 shipped with `package-type` defaulting to `python` just to avoid having to do updates across all the repos to make progress there.

But that it'd be better to be explicit everywhere (like @bdice mentioned in https://github.com/rapidsai/cudf/pull/15483#discussion_r1725767995), to avoid issues like https://github.com/rapidsai/cudf/pull/16650.

Anyway yeah, we *are* adding C++ wheels here very soon so I'll leave this change in here.

Reply:
Just to clarify: I wasn't meaning the passing of `python` to the GHA tool, just the passing of an argument to our script, i.e. we could have hardcoded `python` in this script and added the `package_type` parameter in a follow-up.

Reply:
ohhhh ok yeah got it, fair